pkg/kubelet/pleg/generic.go

```diff
@@ -176,12 +179,12 @@ func (g *GenericPLEG) Healthy() (bool, error) {
 	return true, nil
 }
 
-func generateEvents(podID types.UID, cid string, oldState, newState plegContainerState) []*PodLifecycleEvent {
+func generateEvents(podID types.UID, cid string, oldState, newState plegContainerState, logger klog.Logger) []*PodLifecycleEvent {
```

Comment:
Make it the first parameter, please. That mirrors the convention of passing `ctx` as first parameter.